examples/cargo_without_cmake/src/main.rs

```diff
@@ -28,6 +32,14 @@ fn main() {
         engine.load(&QUrl::from("qrc:/qt/qml/com/kdab/cxx_qt/demo/qml/main.qml"));
     }
 
+    let str = CString::new("test").unwrap();
+
+    let mut context: QMessageLogContext = QMessageLogContext::default();
+    context.set_line(5);
+    context.set_file(&str);
```

Comment:
wonder if we want CString or just a normal Rust string here then convert it behind the scenes?

Reply:
It should be CString, I would like to use String but there's numerous issues. Most notably it doesn't have a null terminator, and there's no good way to get around that. For now a CString is good enough though :)